devnotes/pnor_layout.md

```diff
@@ -0,0 +1,161 @@
+| RINGOVD                     | 19 | 039C9000:039E8FFF  | 20000   | d    | -     | same as size | _unknown_                   |
+| WOFDATA                     | 1A | 039E9000:03CE8FFF  | 300000  | d    | -     | same as size | _unknown_                   |
+| HB_VOLATILE                 | 1B | 03CE9000:03CEDFFF  | 5000    | d    | -     | same as size | _unknown_                   |
+| MEMD                        | 1C | 03cee000:03CFBFFF  | E000    | d    | -     | same as size | _unknown_                   |
+| SBKT                        | 1D | 03d02000:03D05FFF  | 4000    | d    | -     | same as size | _unknown_                   |
+| HDAT                        | 1E | 03d06000:03D0DFFF  | 8000    | d    | -     | same as size | _unknown_                   |
+| UVISOR                      | 1F | 03d10000:03E0FFFF  | 100000  | d    | -     | same as size | _unknown_                   |
+| [BOOTKERNFW](#BOOTKERNFW)   | 20 | 03e10000:03FEFFFF  | 1E0000  | d    | -     | same as size | _unknown_                   |
+| [BACKUP_PART](#BACKUP_PART) | 21 | 03ff7000:03FFEFFF  | 8000    | d    | -     | 0            | _unknown_                   |
+
+Type:
+* l = FFS_TYPE_LOGICAL
+* d = FFS_TYPE_DATA
+* p = FFS_TYPE_PARTITION
+
+Flags:
+* b = FFS_FLAGS_U_BOOT_ENV
+* p = FFS_FLAGS_PROTECTED
+
+# Reproducibility
+Reproducibility of partitions can be found at the
+[wiki page](https://wiki.raptorcs.com/wiki/Firmware/Reproducible_Builds/Status).
+
+# Partition description
+## part
+This partition contains partition table.
+
+## HBEL
+This partition is used by `hostboot` to to save crash information for later
+debug.
+More informations can be found at the
+[wiki page](https://wiki.raptorcs.com/wiki/Hostboot_Debug_Howto).
+
+## GUARD
+This partition is responsible for holding information about components that
+are detected as broken.
+More information about it can be found at the
+[wiki page](https://wiki.raptorcs.com/wiki/Troubleshooting/Guard_Partition).
+
+## SECBOOT
+This partition is used to hold secure variables across renppts in
+tamper-resistant manner. Because writes to PNOR can't be prevented, hash of
+this partition is stored in the `TPM NV`, and verified at the boot time.
+More informations can be found at the
+[wiki page](https://open-power.github.io/skiboot/doc/secvar/secboot_tpm.html).
+
+## HBB
+This partition is overwritten with `coreboot` image containing `bootblock`.
+
+## HBI
+This partition is overwritten with `coreboot` image containing `romstage`,
+`ramstage` and `payload` containing `hostboot`.
+
+## HCODE
+FW for PPE
+
+## PAYLOAD
+This partition contains `skiboot`.
+
+## BOOTKERNEL
+This partition contains `skiroot`.
+
+## VERSION
+Probably just an information about versions of a binaries in the PNOR image.
+
+## HBBL
+This partition is used by hostboot during update operation.
+
+## IMA_CATALOG
+This partition holds a list of events supported by OCC microcode.
+
+## BOOTKERNFW
+Probably `petitboot`, `initramfs`.
```

Comment:
@IgorBagnucki According to the config you have dumped, the initramfs is bundled with kernel image: https://github.com/3mdeb/openpower-coreboot-docs/pull/41/files#diff-927e415b44f07619c187353e91eed8ebfa02a5489973f5c93184653387f8782cR134

So there is a single kernel image which contains the initramfs already. And the initramfs contains the petitboot binaries of course. So all of that is in the `BOOTKERNEL` partition

Reply:
Fixed